src/main/resources/README-CONTACT-ME.md

```diff
@@ -0,0 +1,124 @@
+# Offer setup
+
+## Alias
+
+2021-02-17_websphere_offerings_contact_me
+
+## Setup details
+
+* No, I would prefer to only list my offer through the marketplace and process transactions independently
+
+* How do you want potential customers to interact with this listing offer?
+
+   * Contact me
+   
+### Test drive
+
+* Not checked Enable a test drive
+
+### Customer leads
+
+CRM System connected
+
+* HTTPS Endpoint
+
+* Contact email
+
+   * orourke@ie.ibm.com;edburns@microsoft.com;rezar@microsoft.com
+   
+* HTTPS endpoint URL
+
+   * https://prod-15.westcentralus.logic.azure.com:443/workflows/9024092aefba434c9db98dc2536423f6/triggers/manual/paths/invoke?api-version=2016-06-01&sp=%2Ftriggers%2Fmanual%2Frun&sv=1.0&sig=SggEKOTUpII26Oc6XYVl7nbSgluiYdVeIXkq19LuTtY
+   
+
+### Microsoft 365 integration
+
+* No, my SaaS offer does not integrate with Microsoft Graph
+
+* No, I do not have published Teams apps, Office add-ins, or SharePoint Framework solutions that I would like to link to this SaaS offer
+
+# Properties
+
+## Categories
+
+* Primary category: Compute
+
+   * Subcategory: Application Infrastructure
+
+## Industries
+
+* Checked Apps 
+
+* Not checked Microsoft Clouds for Industry
+
+* App version: blank
+
+## Legal
+
+* Checked Use the Standarard Contract for Microsoft's commercial marketplace?
+
+## Custom amendments...
+
+No content.
+
+# Offer Listing
+
+## Name
+
+IBM WebSphere Product Family on Azure Overview
+
+## Search results summary
+
+The IBM WebSphere product family is a suite of enterprise Java application servers.
+
+## Description (Observed max 2973 characters, not including formatting)
+
+<p>The product family powers some of the most mission critical enterprise applications across the globe and includes WebSphere (Traditional) Application Server, WebSphere Liberty and Open Liberty.</p>
+
+<p>WebSphere products are key components in enabling enterprise Java workloads on Azure. IBM and Microsoft collaborate on a set of jointly developed and supported solutions for the product family. The offers aim to cover a range of use cases from mission critical existing traditional workloads to cloud native applications. Offers include Open Liberty on Azure Red Hat OpenShift (ARO), WebSphere Liberty on ARO, WebSphere Application Server (Traditional) on Virtual Machines, Open Liberty on the Azure Kubernetes Service (AKS), and WebSphere Liberty on AKS. The offers make it as easy as possible to migrate your application to the cloud by automating most boilerplate Azure and Java resource provisioning and configuration tasks. Once initial provisioning is complete, you are completely free to customize deployments further.</p>
```

Comment:
* I've always found the "collaborate on" a bit weird.  For me, "are collaborating on" seems better.
* Rather than "offers aim to cover", how about "offers cover" now that we have a few available.
* I like to hyphenate "cloud-native"
* I'd be tempted to change the order of the offer listings. to ARO, AKS, then VMs.